library/ssl_tls13_keys.c

```diff
@@ -1019,14 +1019,14 @@ int mbedtls_ssl_tls13_populate_transform(
 
 #if !defined(MBEDTLS_USE_PSA_CRYPTO)
     if ((ret = mbedtls_cipher_setkey(&transform->cipher_ctx_enc,
-                                     key_enc, mbedtls_cipher_info_get_key_bitlen(cipher_info),
+                                     (int)key_enc, mbedtls_cipher_info_get_key_bitlen(cipher_info),
```

Comment:
(1) The Mbed TLS coding style now wants a space after a cast
(2) The cast should be on the `mbedtls_cipher_info_get_key_bitlen()` call, not the `key_enc` before it

Reply:
Pff. What a silly mistake I made, fixed now :))